builder/pattern_assembler.js

```diff
@@ -63,7 +63,7 @@
         if (config.debug) {
           console.log('rendering full oPattern: ' + pattern.name);
         }
-        return pattern.render(data, partials);
+        return pattern.render(pattern, data, partials);
```

Comment:
agreed - this seems like a simple mistake given https://github.com/pattern-lab/patternlab-node/blob/pattern-engines/builder/object_factory.js#L55